helix-term/src/application.rs

```diff
@@ -737,11 +737,11 @@ impl Application {
                         }
                         // have to inline the function because of borrow checking...
                         let doc = self.editor.documents.values_mut()
-                            .find(|doc| doc.path().map(|p| p == &path).unwrap_or(false))
+                            .find(|doc| doc.uri().map(|u| u == uri).unwrap_or(false))
```

Comment:
drive by fix: this will trigger a clippy lint in the futur so may aswell do this now
```suggestion
                            .find(|doc| doc.uri().is_some_and(|u| u == uri)
```